site/content/internal/rd-teams.md

```diff
@@ -0,0 +1,67 @@
+---
+title: R&D Teams
+date: 2019-02-06T14:28:35-05:00
+section: internal
+weight: 110
+---
+
+## Leadership
+
+* Corey Hulen - CTO
+* Lindsay Brock - Head of Product
+
+## Apps Team
+
+* Elias Nahum - Eng Lead
+* Harrison Healey - Engineer
+* Saturnino Abril - Engineer
+* Sudheer Timmaraju - Engineer
+* Dean Whillier - Engineer
+* Miguel Alatzar - Engineer
+* Eric Sethna - PM
+* Linda - QA Lead
+
+## Cloud Team
+
+* Joram Wilander - Eng Lead
+* Jesse Hallam - Engineer
+* Carlos Panato - Engineer/DevOps
+* Brad Coughlin - Engineer
+* Gabe Jackson - Engineer
+* Katie Wiersgalla - PM
+
+## Extensibility Team
+
+* Lev Brouk - Eng Lead
+* Christopher Speller - Engineer
+* Jason Blais - PM
+* Dylan Haussermann - QA
+
+## Platform Team
+
+* George Goldberg - Eng Lead
+* Jesus Espino - Engineer
+* Martin Kraft - Engineer
+* Daniel Schalla - Engineer
+* Miguel de la Cruz - Engineer
+* Katie Wiersgalla - PM
+* Lindy Isherwood - QA
+
+## Security Team
+
+* Christopher Speller - Engineer
+* George Goldberg - Eng Lead
+* Daniel Schalla - Engineer
+* Harrison Healey - Engineer
+
+## UX Team
```

Comment:
I've just been speaking with people about trying out having UX designers embedded in dev teams as well. 

To start with, teams will be: 
- Platform: Mike 
- Extensibility: Asaad
- Apps: Andrew

And for Cloud team, Katie will pull from whoever she needs (likely Andrew for end user facing things, Mike for admin things).